src/olympia/editors/helpers.py

```diff
@@ -563,14 +569,10 @@ def process_comment(self):
             self.version.update(**kw)
         self.log_action(amo.LOG.COMMENT_VERSION)
 
-
-class ReviewAddon(ReviewBase):
```

Comment:
Note: It's better to review https://github.com/mozilla/addons-server/pull/5153/commits/44aff0ab2b7c23963290855b75af6f797431a0cc and then https://github.com/mozilla/addons-server/pull/5153/commits/6c1a0f1d3894ccd845eb50b099c25e51a8c63084 separately to understand what was changed in this file.